b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/createRoleComponent/createRoleComponent.js

```diff
@@ -0,0 +1,146 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from "react";
+import { Loader, Modal, Panel, Steps, useToaster } from "rsuite";
+import General from "./createRoleComponentInner/general";
+import Permission from "./createRoleComponentInner/permission";
+import Users from "./createRoleComponentInner/users";
+import styles from "../../../../../../../styles/Settings.module.css";
+import { decodeCreateRole } from "../../../../../../../util/apiDecode/settings/role/deocdeCreateRole";
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from "../../../../../../../util/util/frontendUtil/frontendUtil";
+import { errorTypeDialog, successTypeDialog } from "../../../../../../common/dialog";
+
+export default function CreateRoleComponent(prop) {
+
+    const { open, setOpenCreateRoleModal, session } = prop;
+
+    const [ loadingDisplay, setLoadingDisplay ] = useState(LOADING_DISPLAY_NONE);
+    const [ step, setStep ] = useState(0);
+    const [ displayName, setDisplayName ] = useState("");
+    const [ permissions, setPermissions ] = useState([]);
+    const [ users, setUsers ] = useState([]);
+
+    const toaster = useToaster();
+
+    /**
+     * change the screens to previous and next
+     * 
+     * @param nextStep - next step
+     */
+    const onChange = (nextStep) => {
+        setStep(nextStep < 0 ? 0 : nextStep > 3 ? 3 : nextStep);
+    };
+    const onPrevious = () => onChange(step - 1);
+    const onNext = () => {
+        if (step == 2) {
+
+            onSubmit(displayName, permissions, users);
+        } else {
+
+            onChange(step + 1);
+        }
+    };
+
+    const onDataSubmit = (response) => {
+        if (response) {
+            successTypeDialog(toaster, "Changes Saved Successfully", "Role created successfully.");
+            onCreateRoleModalClose();
+        } else {
+            errorTypeDialog(toaster, "Error Occured", "Error occured while creating the role. Try again.");
+        }
+    };
+
+    const onSubmit = async (displayName, permissions, users) => {
+        setLoadingDisplay(LOADING_DISPLAY_BLOCK);
+        await decodeCreateRole(session, displayName, permissions, users)
+            .then((response) => onDataSubmit(response))
+            .finally(() => setLoadingDisplay(LOADING_DISPLAY_NONE));
+    };
+
+
+    /**
+     *  callback function on create role modal close
+     */
+    const onCreateRoleModalClose = () => {
+        setOpenCreateRoleModal(false);
+        setStep(0);
+        setDisplayName("");
+        setPermissions([]);
+        setUsers([]);
+    };
+
+    const craeteRoleItemDetailsComponent = (currentStep) => {
+
```

Comment:
make this also consistent. leave a line after the declaration.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e535216c72dd635ee40f158f18eab8bdfffa3585